websocket/encoder.go

```diff
@@ -0,0 +1,32 @@
+package websocket
+
+import (
+	"encoding/json"
+)
+
+const (
+	StatusOK    = "ok"
+	StatusError = "error"
+)
+
+type (
+	message struct {
```

Comment:
is this whole thing even used anymore?

Reply:
Yes, only for sending messages to users. but not renamed it to response and moving it to types.